draft-ietf-scitt-architecture.md

```diff
@@ -388,32 +388,36 @@ Registration Policies refer to additional checks over and above the Mandatory Re
 
 Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is eligible for registration.
```

Comment:
I still don't see the utility of requirement to apply registration policies *on registration* for interoperability. Provided we are clear about the difference between a database of transparent statements and the verifiable datastracture that makes them tamper evident, there is no realy value in requiring a check before entry. And this apparent losening actually allows for more powerful retrospective analysis based on transparent statements about policy. Transparent retrospective policies can perfectly well reference the commitments of signed statements to append only logs even after the pre image data has been put beyond use (to counter posoning or satisfy legal requests). 
```suggestion
Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is considered acceptable. These policies may retrospectively apply to previously registered signed statements which have been redacted.```
```suggestion
```

Reply:
Very useful perspective you might want to clean up the code block offset at the bottom for merge.

```suggestion
Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is considered acceptable. These policies may retrospectively apply to previously registered signed statements which have been redacted.
```

Reply:
```suggestion
```
I propose we remove the last remark on retrospective applications of policies. As detailed in the mail thread, this remark generally applies to some other policies that may be supported by the TS, but not to the registration policy as defined in the architecture. If need be, let's open a separate issue or PR to explain them in relation to auditing.

Reply:
yep, agree. especially if we get ories change below (for line 389 -> "Transparency Services MUST maintain Registration Policies.")